src/PythonQt.cpp

```diff
@@ -275,11 +287,14 @@ void PythonQt::init(int flags, const QByteArray& pythonQtModuleName)
       "QtSystemMsg"
     };
     
-    for (auto i = 0u; i<sizeof(enumValues)/sizeof(int); i++) {
+    for (auto i = 0u; i < sizeof(enumValues)/sizeof(int); i++) {
       PyObject* obj = PyInt_FromLong(enumValues[i]);
-      PyModule_AddObject(pack, enumNames[i], obj);
-      Py_INCREF(obj);
-      PyModule_AddObject(pack2, enumNames[i], obj);
+      if (PyModule_AddObject(pack, enumNames[i], obj) >= 0) {
```

Comment:
What >= 0 means here? As far as I understand, 0 is always returned on success.

Reply:
You are right, thank you. I'll improve the readability, this is just "not <" inverted without any thoughts.